### PR TITLE
Code/style fixes

### PR DESCRIPTION
**Pull request by apacheli**
* Remove getters from Member & Role (reverted)
* Destruct properties on imports
* Simplify Base#toJSON() because the properties being checked for are from classes that don't extend Base
* Use Array#fill() on RequestHandler because it's less verbose
* Fix typos

### CODE REVIEW

**Comment by Ratismal** [comment]
Removing every single getter from the member and role objects is a hugely breaking change, and in my opinion should go in it's own PR for discussion.

I don't understand all this hate for getters, anyways. they barely cost anything and offer a lot of convenience.

**Comment by apacheli** [comment]
Alright, I have reverted the getter changes. I'll open another PR later.

**Comment by DonovanDMC** [comment]
removing those getters would break a ton of stuff in my code, so I would rather not have that

File: lib/structures/Base.js [hunk 1]
```diff
@@ -14,19 +14,7 @@ class Base {
...
-        let ID = "";
-        if(this.id) {
-            ID = " " + this.id;
-        } else if(this.user) {
-            ID = " " + this.user.id;
-        } else if(this.baseObject) {
-            ID = "<" + this.baseObject.name + ">";
-        } else if(this.label) {
-            ID = " " + this.label;
-        } else if(this.code) {
```
Comment (abalabahaha) [inline lib/structures/Base.js:26]: Isn't this still used for `Invite`?

**Pull request merged**